packages/jest-worker/src/workers/messageParent.ts

```diff
@@ -8,12 +8,8 @@
 import {PARENT_MESSAGE_CUSTOM} from '../types';
 
 const isWorkerThread = () => {
-  try {
-    const {isMainThread, parentPort} = require('worker_threads');
-    return !isMainThread && parentPort;
-  } catch (_) {
-    return false;
-  }
+  const {isMainThread, parentPort} = require('worker_threads');
```

Comment:
Why have we removed the try/catch here? Is it caught somewhere higher up the call stack?

Reply:
`worker_threads` are available on all supported versions of node

(I assume this is why the check is removed)

Reply:
It seems I was mistaken. I assumed as nodev8 didn't have `worker_threads` support, it didn't allow to "run in parallel". But jest does in fact has its own `ChildProcessWorker` when `worker_threads` is not supported. 

It's slowly coming together. Adding the try/catch again, sorry!